debug-0-starting-balance.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/shardeum/shardeumFlags.ts b/src/shardeum/shardeumFlags.ts
+index 5b212250..b61484e8 100644
+--- a/src/shardeum/shardeumFlags.ts
++++ b/src/shardeum/shardeumFlags.ts
+@@ -278,7 +278,7 @@ export const ShardeumFlags: ShardeumFlags = {
+ 
+   unifiedAccountBalanceEnabled: true,
+   failedStakeReceipt: true,
+-  debugDefaultBalance: '100', //In debug mode the default value is 100 SHM.  This is needed for certain load test operations
++  debugDefaultBalance: '0', //In debug mode the default value is 100 SHM.  This is needed for certain load test operations
```

Comment:
why comment is not fixed ?

Reply:
This is only a patch file not fix described in the PR, but I updated the comment to match the value in the patch @kun6fup4nd4